taxi/urls.py

```diff
@@ -13,38 +13,64 @@
     ManufacturerCreateView,
     ManufacturerUpdateView,
     ManufacturerDeleteView,
+    DriverCreateView,
+    DriverDeleteView,
+    DriverUpdateView,
+    RemoveDriverFromCarView,
+    AssignDriverToCarView,
 )
 
 urlpatterns = [
     path("", index, name="index"),
     path(
         "manufacturers/",
         ManufacturerListView.as_view(),
-        name="manufacturer-list",
+        name="manufacturer_list",
```

Comment:
Code Style issue: Use `-` to split words in URL identification parameter `name`, not the `_`. Please change `manufacturer_list` to `manufacturer-list`.